.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "eslintignore"
+    ]
+}
```

Comment:
This also requires a new line.

Reply:
I also think your `.vscode/settings.json` should be for your workspace and should be included in your `.gitignore` file.